app/models.py

```diff
@@ -1286,10 +1297,17 @@ def status(self, value):
 
     @status.expression
     def status(cls):
+        # To filter by 'awarded' status, we need an explicit EXISTS query on BriefResponse.
+        # This mirrors the awarded_brief_response relationship query above.
         return sql_case([
             (cls.withdrawn_at.isnot(None), 'withdrawn'),
             (cls.published_at.is_(None), 'draft'),
-            (cls.applications_closed_at > datetime.utcnow(), 'live')
+            (
+                exists([BriefResponse.id]).where(
+                    sql_and(cls.id == BriefResponse.brief_id, BriefResponse.awarded_at != None)  # noqa
```

Comment:
We can indeed - I'd tried `is not None` and got an error, but `isnot(None)` works fine. Thanks George!